extensions/vscode/webviews/homeView/src/components/EvenEasierDeploy.vue

```diff
@@ -326,13 +309,40 @@ const deploymentTitle = computed(() => {
   return result.title;
 });
 
-const deploymentSubTitle = computed(() => {
+const deploymentSubTitles = computed(() => {
+  const subTitles: string[] = [];
+  subTitles.push(credentialSubTitle.value);
+  if (entrypointSubTitle.value) {
+    subTitles.push(entrypointSubTitle.value);
+  }
+  return subTitles;
+});
+
+const credentialSubTitle = computed(() => {
   if (home.serverCredential?.name) {
     return `${home.serverCredential.name}`;
   }
   return `Missing Credential for ${home.selectedContentRecord?.serverUrl}`;
 });
 
+const entrypointSubTitle = computed(() => {
+  if (
+    home.selectedConfiguration &&
+    !isConfigurationError(home.selectedConfiguration)
+  ) {
+    const contentRecord = home.selectedContentRecord;
+    const config = home.selectedConfiguration;
+    if (contentRecord) {
+      if (isConfigInError.value) {
+        return `${contentRecord.projectDir}${path.sep}`;
+      } else {
+        return `${contentRecord.projectDir}${path.sep}${config.configuration.entrypoint}`;
+      }
+    }
+  }
+  return "ProjectDir and Entrypoint not determined";
+});
```

Comment:
Since this relies heavily on `home` store data, have you considered using a store getter to handle most of this? 
e.g:
```js
const entrypointSubTitle = computed(() => {
  return home.entryPointSubtitle || "ProjectDir and Entrypoint not determined";
});
```

Reply:
A helper to generate the entrypoint string based on relative path root, the selected configuration, and the path separator seems like a create function. It is just unfortunate that we need it in the home view, and in the extension code. That makes it a bit harder to generalize - although it is still possible.

Reply:
Thanks for the suggestion, but I agree that we want to keep this presentation calculation out of the store.